### PR TITLE
API to communicate with VMS

### LINKED ISSUE #411: API to communicate with VMS
## Description
Create an API to send data of meetups like event name, date and venue to VMS.

### Update [Required]
- [ ] [LIST ITEMS]
### Enhancement to Update [Optional]
- [ ] [LIST ITEMS]

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
[8-9] hours

### PR DESCRIPTION
**Pull request by abhi20161997**
### Description
Created an API to send meetup data  like event name, venue and start date to VMS. Also, added tests for the same to increase coverage.

Fixes #411 

### Type of Change:

- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Drafted the Test Cases for both the get and post views created for the API.
screenshot from 2018-06-11 13-39-02

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by ExactlyMay** [comment]
@abhi20161997 Do Meetups have an end time? Can Meetups be longer than one day?

**Comment by monajalal** [comment]
@ExactlyMay  We could imagine hackathons or learathons that are meetups but longer than one day (like usually 3 days).

File: systers_portal/meetup/tests/test_views.py [hunk 1]
```diff
@@ -1831,3 +1833,53 @@ def check_message_in_response(self, _message_tag=None, _message=None, status_cod
...
             self.assertEqual(message.tags, _message_tag)
             self.assertTrue(_message in message.message)
+
+
+class ApiForVmsViewTestCase(APITestCase, TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username='foo', password='foobar',
+                                             email='user@test.com')
+        self.systers_user = SystersUser.objects.get(user=self.user)
+        country = Country.objects.create(name='Bar', continent='AS')
```
Comment (Monal5031) [inline systers_portal/meetup/tests/test_views.py]: I don't know much about portal, but should this be `self.country`?

Comment (Monal5031) [inline systers_portal/meetup/tests/test_views.py]: I don't know much about portal but should this be `self.country`?

**Comment by anjali-dhanuka** [comment]
@monajalal But that hackathons or learnathons should at least have an end time?

File: systers_portal/meetup/tests/test_views.py (hunk 1 above)
Comment (abhi20161997) [reply systers_portal/meetup/tests/test_views.py]: No, as the field isn't directly used as a field , self doesn't need to be attached .  Also, you can see while testing the meetup location which is the first test in the series, the same format has been used.

**Comment by abhi20161997** [comment]
@sa7i  I changed the redundant snippet to  a single function. Please have a look.

**Pull request merged**